packages/maskbook/src/extension/background-script/CryptoServices/addPerson.ts

```diff
@@ -4,7 +4,7 @@ import { getNetworkWorker } from '../../../social-network'
 import { verifyOthersProve } from './verifyOthersProve'
 import { memoizePromise } from '../../../utils/memoize'
 import { queryPersonaRecord } from '../../../database'
-import type { PersonaRecord } from '../../../database/Persona/Persona.db'
+import type { PersonaRecord } from '@dimensiondev/maskbook-shared'
```

Comment:
I think `queryPersonaRecord` and `queryPersonaRecord` should come from a same or closed place.